pkg/sfu/buffer/dependencydescriptorparser.go

```diff
@@ -0,0 +1,104 @@
+package buffer
+
+import (
+	"fmt"
+
+	dd "github.com/livekit/livekit-server/pkg/sfu/dependencydescriptor"
+
+	"github.com/livekit/protocol/logger"
+)
+
+type videoLayer struct {
```

Comment:
Can we pull out this definition from `sfu/forwarder` and use it across the board. We have something similar there. In the future, we may need a third dimension (quality, i. e. I think SVC codecs like H.264 support a range of qualities for given spatial, temporal layer and that is another dimension that might be used in stream allocator later, would be good to have these kind of common structures in some types module so that we can use it consistently)